src/main/cpp/subsystems/claw.cpp

```diff
@@ -0,0 +1,21 @@
+#include "subsystems/Claw.h"
+
```

Comment:
You need to at least put placeholder functions for `Periodic` and `SimulationPeriodic`